python/tvm/driver/tvmc/composite_target.py

```diff
@@ -21,6 +21,8 @@
 
 from tvm.relay.op.contrib.arm_compute_lib import partition_for_arm_compute_lib
 from tvm.relay.op.contrib.ethosn import partition_for_ethosn
+from tvm.relay.op.contrib.vitis_ai import partition_for_vitis_ai
+from tvm.contrib.target import vitis_ai
```

Comment:
We need to make sure it's imported because it registers the `relay.ext.vitis_ai` codegen: https://github.com/apache/tvm/blob/5d5bbfb1ed2281c3e9b2719be6643281d60022fb/python/tvm/contrib/target/vitis_ai.py#L64

Reply:
Ah I see.